server/views/errorPage/noSearch.handlebars

```diff
@@ -0,0 +1,23 @@
+<div class="container d-flex align-items-center flex-column">
+  <div class="scontainer d-flex align-items-center flex-column">
+    <h1>Unfortunately we did not find any products!</h1>
+    <p>We do not have any products with the description entered.</p>
+    <img
+      class="thumb-image loaded notFound"
+      data-src="https://images.squarespace-cdn.com/content/v1/51cdafc4e4b09eb676a64e68/1470175715831-NUJOMI6VW13ZNT1MI0VB/image-asset.jpeg"
```

Comment:
can you add a //TODO: comment to remind a clean up when we have our stuff next week?

Reply:
I'll add that